src/atom-config/make-handler/lint-html.js

```diff
@@ -0,0 +1,24 @@
+'use babel'
+
+import { spliceReplace1 } from '../../m-utils'
+
+
+const hasHtmlScope = config =>
+  config.scopes.indexOf(config.embeddedScope) !== -1
+
+/* eslint-disable no-param-reassign */
```

Comment:
Never blanket disable a rule without an amazingly good reason. This is definitely not one of those cases. Use a `// eslint-disable-next-line`.

If this function is just directly modifying the passed in `config` then you do need this. If it's returning a _new_ config that is then used you shouldn't even need this disable comment (if not, why are you returning `config`???).

Reply:
see reply under `lint-html.js`